tests/core_tests/config_tests.py

```diff
@@ -0,0 +1,129 @@
+
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2020 - 2023 Pionix GmbH and Contributors to EVerest
```

Comment:
BTW: Is it correct to backdate the copyright? :shrug:

Reply:
removed

Reply:
Looking at the codebase there are pretty much two schools of thought here:

1. Not setting it at all (which is the default for modules created by ev-cli module create btw.)
2. Setting it with a date range starting from project inception to today

I would lean towards 1) here because the date ranges are almost never correctly updated in PRs